src/web/stash/ItemTooltip.tsx

```diff
@@ -60,6 +96,9 @@ export function ItemTooltip({ item }: { item: Item }) {
             Defense:{" "}
             <span class={item.enhancedDefense ? "magic" : ""}>
               {item.defense}
+              <span class="socketed">
+                {" "}[{base["def"][0]} - {base["def"][1]}]
```

Comment:
This isn't the true range, right? If an item has +ac without percentage, then the value displayed for defense will be outside of the range displayed.